skimage/io/manage_plugins.py

```diff
@@ -114,8 +112,9 @@ def _scan_plugins():
 
     for filename in config_files:
         name, meta_data = _parse_config_file(filename)
+        if 'providers' not in meta_data:
```

Comment:
Oh, I see the error: it's not 'providers', it's 'provides' (see line below).

```suggestion
        if 'provides' not in meta_data:
```

Reply:
Great. I'll take a look to see that it actually fixes the issues soon.
Thanks @jni!